samples/powermax_v260_ops_410.yaml

```diff
@@ -82,8 +117,6 @@ spec:
       # Allowed values: map of key-value pairs
       # Default value: None
       nodeSelector:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
Examples for master  under nodeSelector and taints can be kept in 4.10 samples as OCP 4.10 uses K8s 1.23

Reply:
updated node selector .